File: Python/Easy/جمع الأعداد الفردية في مصفوفة.py

# Creator: M7ilan
# https://profile.satr.codes/m7ilan/public/overview
# Discord: M7ilan#5185
# https://github.com/M7ilan

def sumOdd(arr: list[int]) -> int:
    x = []
    for i in arr:
        if i % 2 != 0:
            x.append(i)
    return sum(x)

File: challenges/JavaScript/1. Easy/جمع الأعداد الفردية في مصفوفة.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function sumOdd(arr) {
	x = 0;
	for (let i = 0; i < arr.length; i++) {
		const e = arr[i];
		if (e % 2 != 0) {
			x += e;
		}
	}
	return x;
}
